ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/downloadFile.component.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    ChangeDetectionStrategy,
+    Component, EventEmitter,
+    Input, Output
+} from '@angular/core';
+
+@Component({
+    selector: 'ddp-download-file',
+    template: `
+        <div id="fileDownloadButton" class="fileDownloadButton">
```

Comment:
It looks like our other components externalize the HTML and CSS.  Why are you doing that differently in here?

Reply:
Not all of them externalize the HTML and CSS, as if you don't have a lot there, you are just avoiding having the one extra file only for several lines.

**Examples:**
activityContent.component.ts,
activityQuestion.component.ts,
groupBlock.component.ts,
groupBlockList.component.ts
activityAnswer.component.ts (has internal HTML, even though the HTML looks quite a huge)
activityDateAnswer.component.ts

and so on and so forth :D